lingetic-spring-backend/docker-compose.yml

```
@@ -0,0 +1,14 @@
+    image: postgres:17-alpine
+    environment:
+      POSTGRES_DB: lingetic
+      POSTGRES_USER: ${POSTGRES_USER:?POSTGRES_USER environment variable is not set}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:?POSTGRES_PASSWORD environment variable is not set}
+    ports:
+      - "5432:5432"
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+
+volumes:
+  pgdata:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)